src/ducks/recurrence/api.js

```diff
@@ -0,0 +1,41 @@
+import set from 'lodash/set'
+import flatten from 'lodash/flatten'
+import omit from 'lodash/omit'
+import { dehydrate } from 'cozy-client'
+
+const RECURRENCE_DOCTYPE = 'io.cozy.bank.recurrence'
```

Comment:
It seems to be a new doctype? https://docs.cozy.io/en/cozy-stack/doctype/#add-your-doctype-to-the-store-and-stack

Reply:
I add that to my TODO, thanks